datasource/netcdf_groundwater.py

```diff
@@ -183,157 +173,107 @@ def available_aggregation_vars(self):
         return list(available_known_vars)
 
     def get_available_variables(self):
```

Comment:
```suggestion
    def available_variables(self):
```

De `self.available_subgrid_map_vars()` methode heeft geen "get_" prefix. Even naar de consistentie kijken.